packages/block-library/src/categories/variations.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { category as icon } from '@wordpress/icons';
+
+const variations = [
+	{
+		name: 'terms',
+		title: __( 'Terms List' ),
+		icon,
+		attributes: {
```

Comment:
If you add `isDefault: true` to this variation, then it will resolve the issue with double entries in the inserter. That should make it quite a compelling solution to the problem I like how you handled the Categories vs the rest case.

Reply:
Ah! TIL 😅

Reply:
Done in cd379f75c6d (and 6a8c539de47).